result/dhcpv6client.yaml

```diff
@@ -0,0 +1,147 @@
+openapi: 3.0.3
+info:
+  title: DHCPv6 results models
+  version: ^0.0.0
+components:
+  schemas:
+    Dhcpv6Client.Metrics.Request:
+      description: >-
+        The request to retrieve DHCPv6 per client metrics/statistics.
+      type: object
+      properties:
+        client_names:
+          description: >-
+            The names of DHCPv6 clients to return results for.
+            An empty list will return results for all DHCPv6 client.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Dhcpv6client/properties/name"
+          x-field-uid: 1
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned except for
+            any result_groups. The name of the DHCPv6 client cannot be excluded.
+          type: array
+          items:
+            type: string
+            x-enum:
+              solicits_tx:
+                x-field-uid: 1
+              advertisements_rx:
+                x-field-uid: 2
+              advertisements_ignored:
+                x-field-uid: 3
+              requests_tx:
+                x-field-uid: 4
+              requests_naked:
+                x-field-uid: 5
+              replies_rx:
+                x-field-uid: 6
+              information_requests_tx:
+                x-field-uid: 7
+              renews_tx:
+                x-field-uid: 8
+              rebinds_tx:
+                x-field-uid: 9
+              releases_tx:
+                x-field-uid: 10
+              reconfigures_rx:
+                x-field-uid: 11
+              rapid_commit_solicits_tx:
+                x-field-uid: 12
+              rapid_commit_replies_rx:
+                x-field-uid: 13
+          x-field-uid: 2
+    Dhcpv6Client.Metric:
+      description: >-
+        DHCPv6 per peer statistics information.
+      type: object
+      properties:
+        name:
+          description: >-
+            The name of a configured DHCPv6 client.
+          type: string
+          x-field-uid: 1
+        solicits_tx:
+          description: >-
+            Number of DHCPSOLICIT messages sent.
+          type: integer
+          format: uint64
+          x-field-uid: 2
+        advertisements_rx:
+          description: >-
+            Number of DHCP Advertise messages received.
```

Comment:
Similar format is better e.g. change this to DHCPADVERTISE or DHCP ADVERTISE

Reply:
updated